benchmark/pub_perf.js

```diff
@@ -13,52 +13,49 @@
  * limitations under the License.
  */
 
-"use strict";
+'use strict'
 
-const NATS = require('../lib/nats');
-const nats = NATS.connect();
-const fs = require('fs');
+const NATS = require('../lib/nats')
+const nats = NATS.connect()
+const fs = require('fs')
 
-///////////////////////////////////////
+/// ////////////////////////////////////
```

Comment:
This looks weird, why did it add that?

Reply:
@derekcollison 'spaced-comment' rule:
https://eslint.org/docs/rules/spaced-comment

The standardjs is opinionated like go, and they don't like that :(

Reply:
ok

Reply:
Yes, but you could still replace with another valid comments maybe?
Like:
```
/*
 * Publish Performance
 *
 */
```
or some others in the list of "correct" comments?

Reply:
Or this maybe?
```
//--------------
// Publisher Performance
//---------------
```